layouts/404.html

```diff
@@ -1,3 +1,5 @@
+{{/* This file is in place as an override to the Docsy theme */}}
```

Comment:
Should this be just "This file is in place as an override to Docsy" like most of the other files?

Reply:
Will put a fixup for this in next PR!